pkg/cvo/status.go

```diff
@@ -27,8 +27,7 @@ import (
 
 const (
 	// ClusterStatusFailing is set on the ClusterVersion status when a cluster
-	// cannot reach the desired state. It is considered more serious than Degraded
-	// and indicates the cluster is not healthy.
+	// cannot reach the desired state. It indicates the cluster is not healthy.
```

Comment:
Why can not we say that it probably means one or more operators are in degraded state, rather than saying not healthy?